google-datacatalog-qlik-connector/tests/google/datacatalog_connectors/qlik/scrape/repository_services_api_helper_test.py

```diff
@@ -36,10 +35,11 @@ def test_constructor_should_set_instance_attributes(self):
         self.assertEqual(
             'test-server/qrs',
             attrs['_RepositoryServicesAPIHelper__base_api_endpoint'])
-        self.assertIsNotNone(attrs['_RepositoryServicesAPIHelper__headers'])
+        self.assertIsNotNone(
+            attrs['_RepositoryServicesAPIHelper__common_headers'])
 
     def test_get_full_app_list_should_use_session_to_call_api(self):
```

Comment:
Same here about what is being tested.

Method under test is `self.__helper.get_full_app_list`. This public method responsibility is returning the app list, using the session to call api matters to other layers who are calling this method?

This test case seems to be testing  an implementation detail, adding unnecessary coupling between the tests and source  code.

Let me know if this makes sense to you.

Reply:
Yes, I agree. I've changed the method names and code. WDYT?